src/app/Plans/components/VMMigrationDetails.tsx

```diff
@@ -0,0 +1,217 @@
+      migration.schedule.begin,
+      migration.schedule.end,
+      migration.other.total,
+      migration.other.status,
+      '', // Action column
+    ];
+  };
+
+  const { sortBy, onSort, sortedItems } = useSortState(migrations, getSortValues);
+  const { currentPageItems, setPageNumber, paginationProps } = usePaginationState(sortedItems, 10);
+  React.useEffect(() => setPageNumber(1), [sortBy, setPageNumber]);
+
+  const { selectedItems: expandedVMs, toggleItemSelected: toggleVMsExpanded } = useSelectionState<
+    ITempMockMigration
+  >({
+    items: sortedItems,
+    isEqual: (a, b) => a.id === b.id,
+  });
+
+  const columns: ICell[] = [
+    { title: 'Name', transforms: [sortable, wrappable], cellFormatters: [expandable] },
+    { title: 'Start time', transforms: [sortable] },
+    { title: 'End time', transforms: [sortable] },
+    { title: 'Data copied', transforms: [sortable] },
+    { title: 'Status', transforms: [sortable] },
+    { title: '' },
+  ];
+
+  const rows: IRow[] = [];
+
+  currentPageItems.forEach((migration: ITempMockMigration) => {
+    const isExpanded = expandedVMs.includes(migration);
+
+    let buttonText = '';
+    if (migration.other.status !== 'Ready') {
+      buttonText = 'Cancel';
+    }
+
+    rows.push({
+      meta: { migration },
+      isOpen: isExpanded,
+      // isOpen: true,
+      cells: [
+        migration.id,
+        migration.schedule.begin,
+        migration.schedule.end,
+        `${Math.round(migration.other.copied / 1024)} / ${Math.round(
+          migration.other.total / 1024
+        )} GB`,
+        {
+          title: <PipelineSummary total={0} current={0} status={migration.other.status} />,
+        },
+        {
+          title: buttonText ? (
+            <>
+              <Button variant="secondary" onClick={() => alert('TODO')} isDisabled={false}>
+                {buttonText}
+              </Button>
+              <PlanActionsDropdown />
+            </>
+          ) : null,
+        },
+      ],
+    });
+    rows.push({
+      parent: rows.length - 1,
+      fullWidth: true,
+      cells: [
+        {
+          title: <VMStatusTable vmstatus={migration.status} />,
+          columnTransforms: [classNamesTransform(alignment.textAlignRight)],
+        },
+      ],
+    });
+  });
+
+  return (
+    <>
+      <PageSection variant="light">
+        <Breadcrumb className={`${spacing.mbLg} ${spacing.prLg}`}>
+          <BreadcrumbItem>
+            <Link to={`/plans`}>Migration plans</Link>
+          </BreadcrumbItem>
+          <BreadcrumbItem>{match?.params.planId}</BreadcrumbItem>
+        </Breadcrumb>
+        <Title headingLevel="h1">Migration Details by VM</Title>
+      </PageSection>
+      <PageSection>
+        <Card>
+          <CardBody>
+            <Pagination {...paginationProps} widgetId="migration-vms-table-pagination-top" />
+            <Table
+              aria-label="Migration VMs table"
+              variant={TableVariant.compact}
```

Comment:
I think we can remove the `variant` prop here, this table doesn't look compact in the mocks. The VMStatusTable should stay compact though.